libmattermost.c

```diff
@@ -1326,9 +1328,9 @@ mm_got_teams(MattermostAccount *ma, JsonNode *node, gpointer user_data)
 		JsonObject *team = json_node_get_object(member);
 		
 		const gchar *team_id = json_object_get_string_member(team, "id");
-		const gchar *display_name = json_object_get_string_member(team, "display_name");
+		const gchar *name = json_object_get_string_member(team, "name");
 		

```

Comment:
display_name may change , name (or id in API calls) is the one used in 'permanent' URLs. (small side effect to be fixed later: in Room List still a display_name should appear)